scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/Scheduler.java

```diff
@@ -608,6 +608,19 @@ void changeJobPriority(JobId jobId, JobPriority priority)
     JobId submit(Job job)
             throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException;
 
+    /**
+     *
+     * @param currentJobId id of the already submitted job
+     * @param jobVariables new job variables while is used to replace job variables of the submitted job
+     * @param jobGenericInfos new generic information while is used to replace generic info of the submitted job
+     * @return job id of the newly submitted job
+     * @throws UnknownJobException if <code>currentJobId</code> does not correspond to any submitted job
+     * @throws PermissionException if user cannot access job with <code>currentJobId</code>
+     */
+    JobId reSubmit(JobId currentJobId, Map<String, String> jobVariables, Map<String, String> jobGenericInfos)
+            throws NotConnectedException, UnknownJobException, PermissionException, JobCreationException,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)